app/assets/javascripts/application.js

```diff
@@ -8,7 +8,7 @@
 //= require govuk_publishing_components/components/button
 //= require govuk_publishing_components/components/govspeak
 //
-//= require jquery/dist/jquery
+//= require jquery
```

Comment:
Hmm I think this is caused by Yarn not running. It looks like Rails 7 may have removed the coupling of assets:precompile and yarn install (https://github.com/rails/rails/pull/43641) but haven't seem to have documented in the update guide. I'll see if I can push something up quickly that sorts it.